db/view/view_update_generator.hh

```diff
@@ -50,7 +50,7 @@ using wait_for_all_updates = bool_class<wait_for_all_updates_tag>;
 
 class view_update_generator : public async_sharded_service<view_update_generator> {
 public:
-    static constexpr size_t registration_queue_size = 5;
+    static constexpr size_t registration_queue_size = 1000;
```

Comment:
If you want to disable this for good, init it to maxing, not just some random high number.